client/src/components/application/NonlinearStepper/ProInfos.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Select, Container, FormControl } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import PropTypes from 'prop-types';
+
+const ProInfos = ({ handleFormInput }) => (
+  <>
+    <Container maxWidth="sm">
+      <FormControl variant="outlined" fullWidth>
+        <InputLabel htmlFor="outlined-age-native-simple">
+          How confident are you at understanding English ?
+        </InputLabel>
+        <Select
+          fullWidth
+          labelId="english-understand"
+          label="How confident are you at understanding English ?"
+          name="eConfident"
+          onChange={handleFormInput}
+        >
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
```

Comment:
@alidah0 Don't forget to change these :) and the ones on line 39-41, 57-59, 75-77 too